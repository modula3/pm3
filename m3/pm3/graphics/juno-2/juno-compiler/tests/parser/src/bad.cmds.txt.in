NOT (*
NOT +
SKIP (*
SKIP +
SKIP | (*
SKIP | +
DO (*
DO +
DO SKIP (*
DO SKIP +
DO SKIP OD (*
DO SKIP OD +
SAVE (*
SAVE +
SAVE X (*
SAVE X +
SAVE X IN (*
SAVE X IN +
SAVE X IN SKIP (*
SAVE X IN SKIP +
SAVE X IN SKIP END (*
SAVE X IN SKIP END +
{ (*
{ +
{ SKIP (*
{ SKIP +
{ SKIP } (*
{ SKIP } +
x, (*
x, +
VAR x IN (*
VAR x IN +
x := (*
x := +
x, y.z (*
x, y.z +
VAR x, y IN (*
VAR x, y IN +
VAR x, y IN SKIP (*
VAR x, y IN SKIP +
x, y.z := (*
x, y.z := +
x, y.z := 6 OD
x, y.z := 6 (*
x < (*
x < +
x < y -> (*
x < y -> +
x(y) (*
x(y) OD
x(y) ; (*
x(y) ; +
x < y |
a + (*
VAR x, y.z IN SKIP
VAR x, y.z IN VAR u, v IN SKIP
VAR x, y.z IN u, v := 2, t
