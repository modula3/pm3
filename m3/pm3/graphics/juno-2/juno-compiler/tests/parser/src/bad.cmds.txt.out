Each line you enter will be parsed as a command.

> NOT (*
*** Lex error: Unclosed comment
NOT 

> NOT +
*** Parse error: Found '+'
NOT 

> SKIP (*
*** Lex error: Unclosed comment
SKIP

> SKIP +
*** Parse error: Found '+'
SKIP

> SKIP | (*
*** Lex error: Unclosed comment
SKIP | 

> SKIP | +
*** Parse error: Found '+'
SKIP | 

> DO (*
*** Lex error: Unclosed comment
DO 

> DO +
*** Parse error: Found '+'
DO 

> DO SKIP (*
*** Lex error: Unclosed comment
DO SKIP 

> DO SKIP +
*** Parse error: Found '+'; Expected 'OD'
DO SKIP 

> DO SKIP OD (*
*** Lex error: Unclosed comment
DO SKIP OD

> DO SKIP OD +
*** Parse error: Found '+'
DO SKIP OD

> SAVE (*
*** Lex error: Unclosed comment
SAVE 

> SAVE +
*** Parse error: Found '+'; Expected '<Identifier>'
SAVE 

> SAVE X (*
*** Lex error: Unclosed comment
SAVE X 

> SAVE X +
*** Parse error: Found '+'; Expected 'IN'
SAVE X 

> SAVE X IN (*
*** Lex error: Unclosed comment
SAVE X IN 

> SAVE X IN +
*** Parse error: Found '+'
SAVE X IN 

> SAVE X IN SKIP (*
*** Lex error: Unclosed comment
SAVE X IN SKIP 

> SAVE X IN SKIP +
*** Parse error: Found '+'; Expected 'END'
SAVE X IN SKIP 

> SAVE X IN SKIP END (*
*** Lex error: Unclosed comment
SAVE X IN SKIP END

> SAVE X IN SKIP END +
*** Parse error: Found '+'
SAVE X IN SKIP END

> { (*
*** Lex error: Unclosed comment
{ 

> { +
*** Parse error: Found '+'
{ 

> { SKIP (*
*** Lex error: Unclosed comment
{ SKIP 

> { SKIP +
*** Parse error: Found '+'; Expected '}'
{ SKIP 

> { SKIP } (*
*** Lex error: Unclosed comment
{ SKIP }

> { SKIP } +
*** Parse error: Found '+'
{ SKIP }

> x, (*
*** Lex error: Unclosed comment
x, 

> x, +
*** Parse error: Found '+'; Expected '<Identifier>'
x, 

> VAR x IN (*
*** Lex error: Unclosed comment
VAR x IN 

> VAR x IN +
*** Parse error: Found '+'
VAR x IN 

> x := (*
*** Lex error: Unclosed comment
x := 

> x := +
*** Parse error: Found '+'
x := 

> x, y.z (*
*** Lex error: Unclosed comment
x, y.z 

> x, y.z +
*** Parse error: Found '+'; Expected ':='
x, y.z 

> VAR x, y IN (*
*** Lex error: Unclosed comment
VAR x, y IN 

> VAR x, y IN +
*** Parse error: Found '+'
VAR x, y IN 

> VAR x, y IN SKIP (*
*** Lex error: Unclosed comment
VAR x, y IN SKIP 

> VAR x, y IN SKIP +
*** Parse error: Found '+'; Expected 'END'
VAR x, y IN SKIP 

> x, y.z := (*
*** Lex error: Unclosed comment
x, y.z := 

> x, y.z := +
*** Parse error: Found '+'
x, y.z := 

> x, y.z := 6 OD
*** Parse error: Found 'OD'
x, y.z := 6

> x, y.z := 6 (*
*** Lex error: Unclosed comment
x, y.z := 6

> x < (*
*** Lex error: Unclosed comment
x < 

> x < +
*** Parse error: Found '+'
x < 

> x < y -> (*
*** Lex error: Unclosed comment
x < y -> 

> x < y -> +
*** Parse error: Found '+'
x < y -> 

> x(y) (*
*** Lex error: Unclosed comment
x(y) 

> x(y) OD
*** Parse error: Found 'OD'
x(y)

> x(y) ; (*
*** Lex error: Unclosed comment
x(y); 

> x(y) ; +
*** Parse error: Found '+'
x(y); 

> x < y |
*** Parse error: Found '|'; Expected '->'
x < y 

> a + (*
*** Lex error: Unclosed comment
a + 

> VAR x, y.z IN SKIP
*** Parse error: Found '.'; Expected 'IN'
VAR x, y 

> VAR x, y.z IN VAR u, v IN SKIP
*** Parse error: Found '.'; Expected 'IN'
VAR x, y 

> VAR x, y.z IN u, v := 2, t
*** Parse error: Found '.'; Expected 'IN'
VAR x, y 

> ^D
